receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/resource.go

```diff
@@ -32,19 +32,19 @@ func containerResource(cm ContainerMetadata) pdata.Resource {
 
 func taskResource(tm TaskMetadata) pdata.Resource {
 	resource := pdata.NewResource()
-	resource.Attributes().UpsertString(AttributeECSCluster, tm.Cluster)
+	resource.Attributes().UpsertString(AttributeECSCluster, getResourceFromARN(tm.Cluster))
```

Comment:
Shouldn't some unit tests need to be added / updated for this change?

Reply:
Yea, its using the an existing function which I renamed. Modified existing unit tests to call this func and test possible cases.

Reply:
Added a new test for this. Thanks.